java-frontend/src/main/java/org/sonar/java/se/checks/OptionalGetBeforeIsPresentCheck.java

```diff
@@ -47,15 +47,7 @@
     "NoSuchElementException will be thrown when invoking method %s() without verifying Optional parameter.");
 
   private enum OptionalConstraint implements Constraint {
-    PRESENT, NOT_PRESENT;
-
-    @Override
-    public String valueAsString() {
```

Comment:
and in order to test it I would have to change its visibility. So I don't see the point of keeping something unused and badly designed.